### DISCUSSION TITLE
Confused with dateformat in webhook

### DISCUSSION THREAD

**Discussion by yizhixiaotaozi** [prometheus/alertmanager#3116] [Q&A] [answered]
I install alertmanager-0.24.0.linux-amd64, and try to receive webhook with a java application.
But the dateformat seems to be unstable.
startsAt is ok, it has the same value as request param to alertmanager,  but endsAt provides different format。
0001-01-01T00:00:00Z         when firing
2021-11-15T00:08:07.123Z    i see in github issuses
2022-10-25T18:55:45.926984246+08:00  when auto resolevd 

The last one is unparseable in java!
please help, thks

**Comment by simonpasquier** [comment, marked as answer]
`2022-10-25T18:55:45.926984246+08:00` is represented in [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339) format.

**Comment by yizhixiaotaozi** [comment]
thanks
